[2023-07-20T14:50:26.702+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.dag_envio_alerta manual__2023-07-20T14:49:21.867555+00:00 [queued]>
[2023-07-20T14:50:26.793+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.dag_envio_alerta manual__2023-07-20T14:49:21.867555+00:00 [queued]>
[2023-07-20T14:50:26.794+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-20T14:50:26.808+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): dag_envio_alerta> on 2023-07-20 14:49:21.867555+00:00
[2023-07-20T14:50:26.812+0000] {standard_task_runner.py:57} INFO - Started process 240 to run task
[2023-07-20T14:50:26.815+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_stock_data_google_DAG', 'dag_envio_alerta', 'manual__2023-07-20T14:49:21.867555+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/ETL_stock_data_google_DAG.py', '--cfg-path', '/tmp/tmp7__9a3r_']
[2023-07-20T14:50:26.817+0000] {standard_task_runner.py:85} INFO - Job 94: Subtask dag_envio_alerta
[2023-07-20T14:50:26.860+0000] {task_command.py:410} INFO - Running <TaskInstance: ETL_stock_data_google_DAG.dag_envio_alerta manual__2023-07-20T14:49:21.867555+00:00 [running]> on host 90317fb99301
[2023-07-20T14:50:26.933+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fvinciguerra' AIRFLOW_CTX_DAG_ID='ETL_stock_data_google_DAG' AIRFLOW_CTX_TASK_ID='dag_envio_alerta' AIRFLOW_CTX_EXECUTION_DATE='2023-07-20T14:49:21.867555+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-20T14:49:21.867555+00:00'
[2023-07-20T14:50:26.935+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-20T14:50:26.936+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/pyscripts/Alert_monthvalues_stock_data_google.py']
[2023-07-20T14:50:26.943+0000] {subprocess.py:86} INFO - Output:
[2023-07-20T14:50:28.092+0000] {subprocess.py:93} INFO - WARNING:root:/opt/***/logs/scheduler/latest already exists as a dir/file. Skip creating symlink.
[2023-07-20T14:50:28.265+0000] {subprocess.py:93} INFO - [[34m2023-07-20T14:50:28.264+0000[0m] {[34mAlert_monthvalues_stock_data_google.py:[0m93} ERROR[0m - Error: name 'host' is not defined[0m
[2023-07-20T14:50:28.333+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-20T14:50:28.358+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=ETL_stock_data_google_DAG, task_id=dag_envio_alerta, execution_date=20230720T144921, start_date=20230720T145026, end_date=20230720T145028
[2023-07-20T14:50:28.436+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-20T14:50:28.452+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
